site/static/js/quiz.js

```diff
@@ -1,57 +1,99 @@
-function selectQuizOption(selectedId) {
-    const currentLevel = selectedId.split('/').length - 1;
-    $('.option-row').each(function (i) {
-        const rowId = $(this).attr('data-quiz-id');
-        // don't hide option rows if it has a lower level
-        // e.g. when clicking "x86_64" under Linux, we don't want to hide the operating system row
-        if ($(this).attr('data-level') < currentLevel) {
-            return;
-        }
-        if (rowId === selectedId) {
-            $(this).removeClass('hide');
-            $(this).find('.option-button').removeClass('active');
-            return;
-        }
-        // hide all other option rows
-        $(this).addClass('hide');
-    });
-    // hide other answers
-    $('.quiz-instruction').addClass('hide');
-    // show the selected answer
-    $('.quiz-instruction[data-quiz-id=\'' + selectedId + '\']').removeClass('hide');
-
-    const buttons = $('.option-row[data-quiz-id=\'' + selectedId + '\']').find('.option-button');
-    // auto-select the first option for the user, to reduce the number of clicks
-    if (buttons.length > 0) {
-        const btn = buttons.first();
-        btn.addClass('active');
-        selectQuizOption(btn.attr('data-quiz-id'));
+function selectQuizOption(selectedId, autoselect = true) {
+  const currentLevel = selectedId.split("/").length - 1;
+  $(".option-row").each(function (i) {
+    const rowId = $(this).attr("data-quiz-id");
+    // don't hide option rows if it has a lower level
+    // e.g. when clicking "x86_64" under Linux, we don't want to hide the operating system row
+    if ($(this).attr("data-level") < currentLevel) {
+      return;
+    }
+    if (rowId === selectedId) {
+      $(this).removeClass("hide");
+      $(this).find(".option-button").removeClass("active");
+      return;
+    }
+    // hide all other option rows
+    $(this).addClass("hide");
+  });
+  // hide other answers
+  $(".quiz-instruction").addClass("hide");
+  // show the selected answer
+  $(".quiz-instruction[data-quiz-id='" + selectedId + "']").removeClass("hide");
+
+  const buttons = $(".option-row[data-quiz-id='" + selectedId + "']").find(
+    ".option-button"
+  );
+  // auto-select the first option for the user, to reduce the number of clicks
+  if (buttons.length > 0) {
+    if (autoselect) {
```

Comment:
Once last thing, this is unnecessarily double nested, could these two be `&&` together